financialaid/views_test.py

```diff
@@ -204,13 +208,13 @@ def test_review_financial_aid_view_with_filter_and_sorting(self):
         FinancialAidFactory.create(tier_program=self.tier_programs["0k"], status=FinancialAidStatus.AUTO_APPROVED)
```

Comment:
Can we add more `FinancialAid`s with the same status? With one element for each we can't verify sort